pandas/tests/io/parser/test_encoding.py

```diff
@@ -272,6 +272,26 @@ def test_chunk_splits_multibyte_char(all_parsers):
     tm.assert_frame_equal(dfr, df)
 
 
+@skip_pyarrow
+def test_readcsv_memmap_utf8(all_parsers):
```

Comment:
should this be a test or an asv?

Reply:
I think there should be both. The patch changes the type of the return value of the `read` function from `str`  to `bytes`. As the `read` function is called from C parser code and the return value is processed by Cython magic string function, to me it wasn't obvious that this will work correctly. If you think the test is not necessary, I can remove it.
I am working on asv test, hope I'll have it very soon.